### DISCUSSION TITLE
Make examples double clickable as well as single click on website

### DISCUSSION THREAD

**Discussion by willingc** [pyscript/pyscript#1197] [Proposals]
The website looks awesome!

This is a minor usability thing on the https://pyscript.net/examples/ page. I tried to double click the example and got no response. Finally figured out a single click activates.

**Comment by JeffersGlass** [comment]
Thanks for submitting this! Looking into it, it seems like part of the issue is that hovering/mousing-over each example highlights the whole rectangular "button" in yellow, but only the title is actually clickable as a link... something we should probably tweak.

**Reply by willingc** [reply]
Thanks for looking into it @JeffersGlass. I appreciate it. Keep up the awesome work.

**Comment by JeffersGlass** [comment]
I've opened a [linked issue](https://github.com/pyscript/pyscript.net/issues/40) over in the pyscript.net repository, where we track issues/feedback related to the website itself.
